### DISCUSSION TITLE
installation

### DISCUSSION THREAD

**Discussion by glitch-txs** [vanilla-extract-css/vanilla-extract#1272] [Q&A] [answered]
shouldn't this be `npm i -D @vanilla-extract/css` ?
for development only?
![image](https://github.com/vanilla-extract-css/vanilla-extract/assets/66949816/9c127b37-20d0-4915-905c-6d9d972bc023)

**Comment by askoufis** [comment, marked as answer]
Practically, your bundler integration should ensure that there are no references to `@vanilla-extract/css` at runtime. Technically you are correct in that `@vanilla-extract/css` is only used during development in app, but this isn't necessarily the case with a library, for example.
